package.json

```diff
@@ -30,5 +30,9 @@
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
     "prepublish": "npm run build"
+  },
+  "dependencies": {
```

Comment:
Yeah the pattern I've been using is to make react a peer and a dev dependency. Can you swap it to that?